plugins/system/accessibility/accessibility.xml

```diff
@@ -17,7 +17,7 @@
 		<language tag="en-GB">language/en-GB/plg_system_accessibility.sys.ini</language>
 	</languages>
 	<config>
-		<fields name="params">
+		<fields name="params" addfieldprefix="Joomla\Component\Menus\Administrator\Field">
```

Comment:
I think the `addfieldprefix` should go inside the `modal_menu` field

Reply:
Done